torch/onnx/symbolic_opset12.py

```diff
@@ -92,3 +93,59 @@ def ge(g, input, other):
 
 def le(g, input, other):
     return g.op('LessOrEqual', input, other)
+
+@parse_args('v', 'i', 'v', 'v')
+def unfold(g, input, dimension, size, step):
+    if sym_help._operator_export_type == torch.onnx.OperatorExportTypes.ONNX_ATEN_FALLBACK:
+        return g.op("ATen", input, operator_s="unfold", dimension_i=dimension, size_i=size, step_i=step)
+    sizes = sym_help._get_tensor_sizes(input)
+    try:
```

Comment:
Could you explain why the try catch is needed here? Isn't there any other way to check `sizes`?

Reply:
checkout `_get_tensor_dim_size` in symbolic_helper.py